db/patterns/mopub.eno

```diff
@@ -1,7 +1,7 @@
 name: MoPub
 category: advertising
 website_url: https://www.mopub.com/
-organization: twitter
+organization: X Corp
```

Comment:
Sorry, not obvious, but for all patterns, the organization must stay at `twitter`. Here, twitter is used as an ID and is the internal link to the organization entry.

(If you change the organization fields back, it should also fix the tests.)